web/concrete/src/User/User.php

```diff
@@ -125,10 +127,10 @@ public function getUserInfoObject()
 
     public function __construct()
     {
-        $app = Application::getFacadeApplication();
+        $this->app = Application::getFacadeApplication();
```

Comment:
It's true that we have fewer calls, but function calls are relatively cheap. `Application::getFacadeApplication()` simply returns a static property, so I'd expect no real change with speed. It's a negligable amount of more RAM, but it does make every `User` subclass have to now have to keep a reference to `Application` which I think is a step in the wrong direction for negligible gains.